### PR TITLE
Add instructions for creating a minidump and adding it to the test suite

### PR DESCRIPTION
**Pull request by hlinnaka**
I patched these instructions together from information that @xinzweb sent me over a private email. Let's put it in the README, so that I know what to do the next time I need to add a test case ;-).

### CODE REVIEW

**Comment by vraghavan78** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -95,6 +95,57 @@ To run a specific minidump, for example for `../data/dxl/minidump/TVFRandom.mdp`
...
+$ ls -l ~/data-master/minidumps/
+total 12
+-rw------- 1 heikki heikki 10818 Jun 10 22:02 Minidump_20160610_220222_4_14.mdp
+   ```
+
+3. Run xmllint on the minidump to format it better, and copy it under the
+   dxl/data/minidump directory:
+
+   ```
+xmllint --format ~/data-master/minidumps/Minidump_20160610_220222_4_14.mdp > dxl/data/minidump/MyTest.xml
```
Comment (danielgustafsson) [inline README.md]: The diff below has `data/dxl/` where it here has `dxl/data/`, I assume the former is the right one?

Comment (hsyuan) [inline README.md]: Yes, the former is right.

**Comment by hlinnaka** [comment]
Fixed the path and pushed. Thanks!

**Pull request closed without merging**